changelog.d/20240520_113001_danfuchs_notebook_directories.md

```diff
@@ -0,0 +1,10 @@
+<!-- Delete the sections that don't apply -->
+
+### Backwards-incompatible changes
+
+- NotebookRunner business now runs all notebooks in a repo by default except in the `mobu_exclude` or its descendants
+- Add `exclude_dirs` option to NotebookRunner business to list additional directories in which notebooks will not be run
+
+### Other changes
+
+- Updated python dependencies
```

Comment:
We don't note this in the change log because we do it with every release, so it just adds noise. Instead, we document at the start of CHANGELOG.md that we do this with every release.